### PR TITLE
Adding the Audit and Traceability to the Plotly dashboard

### PR DESCRIPTION
**Pull request by dianajlailaty**
In this PR, I updated the documentation related to the Audit and Traceability page

### CODE REVIEW

**Review by andrewssobral — APPROVED** [review]

File: src/docs/PML/PMLUserGuide.adoc [hunk 1]
```diff
@@ -797,23 +797,33 @@ image::MAAS_ML_Delete_Service.PNG[align=center]
...
+** Audit and Traceability
+** Dataset Analytics
+** Data Drift Analytics
+** Predictions Preview
+
+===== Audit and Traceability
+
+
+When clicking on the endpoint, the user is redirected to a 4 tabs webpage. By default, the Audit and
+Traceability tab opens in which the user can check the different chosen values of the MaaS_ML instance
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:816]: to refactor: the Audit and Traceability tab opens in which ....

File: src/docs/PML/PMLUserGuide.adoc [hunk 2]
```diff
@@ -797,23 +797,33 @@ image::MAAS_ML_Delete_Service.PNG[align=center]
 
 There is also one more action that can be executed from Service Automation Portal which is:
 
-- *Update_MaaS_ML_Parameters*: This action enables you to update the variables values associated to the MaaS_ML instance according to your new preferences.
+- *Update_MaaS_ML_Parameters*: This action enables you to update the variables values associated
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:800]: variable values

File: src/docs/PML/PMLUserGuide.adoc [hunk 3]
```diff
@@ -797,23 +797,33 @@ image::MAAS_ML_Delete_Service.PNG[align=center]
...
+** Dataset Analytics
+** Data Drift Analytics
+** Predictions Preview
+
+===== Audit and Traceability
+
+
+When clicking on the endpoint, the user is redirected to a 4 tabs webpage. By default, the Audit and
+Traceability tab opens in which the user can check the different chosen values of the MaaS_ML instance
+variables. In addition, the traceability information and warnings are listed in a table where each row represents
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:817]: each row represents
the initialization, deployment, prediction etc... information 
to refactor like infrormation about the initi.., dep, pred, etc

**Comment by mboussaa** [comment]
@dianajlailaty some minor comments, and then could you please the line breaks? Thanks. Please, ping when done, I will review again. Thx

File: src/docs/PML/PMLUserGuide.adoc [hunk 4]
```diff
@@ -841,24 +841,28 @@ image::MAAS_ML_Delete_Service.PNG[align=center]
...
 
+** Audit and Traceability
 ** Dataset Analytics
 ** Data Drift Analytics
 ** Predictions Preview
 
+===== Audit and Traceability
+
+When clicking on the endpoint, the user is redirected to a 4 tabs webpage. By default, the Audit and
+Traceability tab opens. In this page, the user can check the different chosen values of the MaaS_ML instance
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:859]: is opened?

File: src/docs/PML/PMLUserGuide.adoc [hunk 5]
```diff
@@ -841,24 +841,28 @@ image::MAAS_ML_Delete_Service.PNG[align=center]
...
+variables. In addition, the MaaS_ML traceability information and warnings are listed in a table where each row represents information about
+the initialization, deployment, prediction etc... corresponding to different date/time(s).
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:861]: , prediction etc... => , prediction, etc.

**Review by mboussaa — APPROVED** [review]

**Pull request merged**